operations/toc-templates/template-dd-pr-graduation.md

```diff
@@ -182,7 +182,14 @@ Note: this section may be augmented by the completion of a Governance Review fro
 
 <!-- (TOC Evaluation goes here) --> 
 
-- [ ] **Document the project's release process.**
+- [ ] **Document the project's release process and guidelines publicly in a RELEASES.md or equivalent file that defines:** 
+
+  - [ ] release expectations (scheduled or based on feature implementation)
+  - [ ] tagging as stable, unstable, and security related releases
+  - [ ] information on branch and tag strategies
+  - [ ] branch and platform support and length of support
+  - [ ] artifacts included in the release.
```

Comment:
Can release artifacts be just the source code tagged in Git, so users would need to build their own distribution to make use of the project?

If we don't want to enforce ready-to-run release artifacts for graduated projects, then I think we should require for projects to provide all the needed documentation on how users can build ready-to-run artifacts from the stable tags. For command line tools, this could be just a `go install`  instruction. For more complex projects like Kubernetes controllers, the documentation should also include the deployment configuration besides the build instructions.